lib/index.js

```diff
@@ -81,11 +82,10 @@ function onThrottledResize() {
 
 function saveSettings() {
   return new Promise((resolve, reject) => {
-    fs.writeFile(settingsFile, JSON.stringify(settings, null, '  '), (err) => {
+    fs.writeFile(settingsFile, yaml.dump(settings), (err) => {
       if (err) {
         return reject(err);
       }
-
       resolve();
```

Comment:
Keep separation between code groups.
```suggestion

      resolve();
```

For example:
```js
function calculate() {
  const a = 1;
  const b = 2;
  let c = 0;

  if (a == 1) {
    c = 3;
  }

  console.log("Results:");
  console.log(`a = ${a}`);
  console.log(`b = ${b}`);
  console.log(`c = ${c}`);
```
In this function, we have 3 different tasks:
1. Set up the variables
2. Modify a variable when certain conditions are met
3. Display the results

It improves readability when the code is separated.

Code blocks (IF, IF-ELSE, SWITCH-CASE, etc) should always be separated from other code.

You should always respect the code style of the project, but in this case having a linter and Husky setup would have avoided these styling issues. I'll setup a linter when I have time.

Reply:
sure, will keep this in mind.